### DISCUSSION TITLE
Include `irc.chat.twitch.tv` on the JOIN rate limiting pool

### DISCUSSION THREAD

**Discussion by ALazyMeme** [Chatterino/chatterino2#3539] [Ideas]
I was looking to reduce my resource usage by moving some of the normal twitch channels over to irc instead (as this would also stop the message history from loading and stop emotes from loading), however I noticed that it will still try to join those irc channel at a normal JOIN limit; which rate limits my normal JOINs.

I would like to see some solution of any servers matching `irc.chat.twitch.tv`, that their rate limits are synced with that of normal chat (somehow, this may be a JOINs only solution). This would also assist for users that may want to farm subs.

**Comment by pajlada** [comment]
This feels like the wrong way about solving performance/resource issues.

Make separate issues for the performance/resource issues instead and we can see what sort of solutions we can think about.
